docs/openstack/keystone_adoption.md

```diff
@@ -54,19 +54,14 @@
   export OS_CLIENT_CONFIG_FILE=clouds-adopted.yaml
   export OS_CLOUD=adopted
 
+  openstack endpoint list | grep ' keystone ' | grep ' admin ' | awk '{ print $2; }' | xargs openstack endpoint delete || true
```

Comment:
I'm ok with this, but at first glance it can be optmized w/ something like:

```
openstack endpoint list | grep keystone | awk '/admin/{ print $2; }' | xargs openstack endpoint delete || true
```

Reply:
Thanks, updating :)